WordPress/Classes/ViewRelated/Gutenberg/GutenbergViewController+MoreActions.swift

```diff
@@ -15,7 +15,7 @@ extension GutenbergViewController {
         /*if mode == .richText {
             // NB : This is a candidate for plurality via .stringsdict, but is limited by https://github.com/wordpress-mobile/WordPress-iOS/issues/6327
             let textCounterTitle = String(format: NSLocalizedString("%li words, %li characters", comment: "Displays the number of words and characters in text"), richTextView.wordCount, richTextView.characterCount)
-            
```

Comment:
Can we remove this whitespace change?

Reply:
I can't seem to whenever I introduce and save it auto-strips. Should it even be there though as no other empty lines have white space at the beginning like that

Reply:
Hmmm, good point. It's not ideal to have this in an unrelated PR, but it's even less ideal to have it in the code in the first place. I'm ok to remove it!